components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/JWTTokenIssuer.java

```diff
@@ -115,6 +119,92 @@ public JWTTokenIssuer() throws IdentityOAuth2Exception {
         signatureAlgorithm = mapSignatureAlgorithm(config.getSignatureAlgorithm());
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String subjectToken(OAuthAuthzReqMessageContext oauthAuthzMsgCtx) throws IdentityOAuth2Exception {
```

Comment:
Wemay need to modify the method name. The method name should denote an action not a noun.

Reply:
resolved with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2460/commits/ea1e6909f7fda24d77ce70bc69c9892ec5d43c23